openem-ingestor/service/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"openem-ingestor/core"
+
+	"github.com/spf13/viper"
+)
+
+func main() {
+	if err := core.ReadConfig(); err != nil {
+		panic(fmt.Errorf("Failed to read config file: %w", err))
+	}
+	log.Printf("Config file used: %s", viper.ConfigFileUsed())
+	log.Println(viper.AllSettings())
+
+}
```

Comment:
good that this is a separate package, the REST API shouldn't depend from Wails. I also like that common functionality could be put into the "core" package